src/fmu/dataio/export/rms/volumetrics.py

```diff
@@ -0,0 +1,253 @@
+        """Convert table to pandas dataframe"""
+        _logger.debug("Read values and convert to pandas dataframe...")
+        dict_values = (
+            self.project.volumetric_tables[self._volume_table_name]
+            .get_data_table()
+            .to_dict()
+        )
+        _logger.debug("Dict values are: %s", dict_values)
+        self._dataframe = dfr = pd.DataFrame.from_dict(dict_values)
+        dfr.rename(columns=_RENAME_COLUMNS_FROM_RMS, inplace=True)
+        dfr.drop("REAL", axis=1, inplace=True, errors="ignore")
+        _logger.debug("Read values and convert to pandas dataframe... DONE")
+
+    def _set_units(self) -> None:
+        """See if the RMS project is defined in metric or feet."""
+
+        units = self.project.project_units
+        _logger.debug("Units are %s", units)
+        self._units = str(units)
+
+    def _read_includes_etc(self) -> None:
+        """Handle other products (placeholder; in prep in the code)."""
+        if self.include_zone_maps:
+            raise NotImplementedError(
+                "Including zone maps is currently not implemented"
+            )
+        if self.include_total_maps:
+            raise NotImplementedError(
+                "Including total maps is currently not implemented"
+            )
+        if self.include_3dgrid_properties:
+            raise NotImplementedError(
+                "Including 3D parameters is currently not implemented"
+            )
+
+    def _warn_if_forcefolder(self) -> None:
+        if self.forcefolder:
+            warn(
+                "A 'forcefolder' is set. This is strongly discouraged and will be "
+                "removed in coming versions",
+                FutureWarning,
+            )
+
+    def _process_data(self) -> None:
+        """Process all routines ^ except exports."""
+        _logger.debug("Process data...")
+        self._check_rmsapi_version()
+        self._set_global_config()
+        self._rms_volume_job_settings()
+        self._read_volume_table_name_from_rms()
+        self._voltable_as_dataframe()
+        self._set_units()
+        self._read_includes_etc()
+        self._warn_if_forcefolder()
+        _logger.debug("Process data... DONE")
+
+    def _export_volume_table(self) -> dict[str, str]:
+        """Do the actual volume table export using dataio setup."""
+
+        edata = dio.ExportData(
+            config=self._global_config,
+            content="volumes",
+            unit="m3" if self._units == "metric" else "ft3",
+            vertical_domain={"depth": "msl"},
+            workflow=self.workflow,
+            forcefolder=self.forcefolder,
+            classification=self.classification,
+            tagname=self.tagname,
+            name=self.grid_name,
+        )
+
+        out = edata.export(self.dataframe)
+        _logger.debug("Volume result to: %s", out)
+        return {"volume_table": out}
+
+    def export(self) -> dict[str, str]:  # public function
```

Comment:
```suggestion
    def export(self) -> dict[str, str]: 
```
I think the comment here is unneeded